### DISCUSSION TITLE
AUP#CNCT

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2431] [Help with using Postal]
Please I would love to know what CNCT means in AUP#CNCT , I would really appreciate any help, knowing what CNCT means in AUP#CNCT  would give me a more insight about what the problem is and how to fix this. Thanks

**Comment by sinnbeck** [comment]
A quick google tells me
>  AUP#CNCT which is too many concurrent connections

But perhaps give some more information on when you are getting the error (I assume its a bounce message?)

Edit: Also found https://www.cox.com/residential/support/email-error-codes.html
This says more or less the same
> There is a limit to the number of concurrent SMTP connections per IP address to protect the systems against attack. Ensure that the sending email server is not opening more than 10 concurrent connections to avoid reaching this limit.

**Comment by willpower232** [comment]
I don't think I've seen that message, I presume it is provider specific. Could you provide a screenshot if the previous post wasn't enough to help?

**Comment by Itseric100** [comment]
![867763BF-BDAF-4C2C-817E-7655733EA835](https://user-images.githubusercontent.com/96216385/235316308-a9e79ef9-58be-4f29-9814-87cda03f2072.jpeg)
@willpower232 @sinnbeck Thank you so much, this happens when I send about 6 emails to @tmomail.net, it happens on the 6th email and onward and I do not know how to fix that. And yes it is a soft fail and a bounce message, I have attached a screenshot of the error . Thank you

**Reply by willpower232** [reply]
421 is supposed to indicate a temporary error which is why it is a soft fail and the message is probably retried?

If you're able to send through a few messages without issue then I guess you could slow down how they're being sent but that depends on how you're sending the messages.
